internal/util/get/get_test.go

```diff
@@ -660,3 +661,19 @@ func TestCommand_DefaultValues_AtVersion(t *testing.T) {
 	c = Command{Git: kptfile.Git{Repo: "foo", Directory: "/", Ref: "r"}}
 	assert.EqualError(t, c.DefaultValues(), "must specify destination")
 }
+
+func TestCommand_Run_ignorefile(t *testing.T) {
+	kyamlext.GetIgnoreFileName = func() string {
+		return kptfile.KptIgnoreFileName
+	}
+	dest := "my-dataset"
+	g, _, clean := testutil.SetupRepoAndWorkspace(t, testutil.DatasetWithIgnoreFile)
+	defer clean()
+
+	err := Command{
+		Git:         kptfile.Git{Repo: g.RepoDirectory, Ref: "master", Directory: "/"},
+		Destination: dest,
+		AutoSet:     true,
+	}.Run()
+	assert.NoError(t, err)
```

Comment:
What does no error mean here? Shouldn't we assert the result against an expected dataset?